tests/gcm/test_stochastic_models.py

```diff
@@ -43,11 +43,11 @@ def test_when_fitting_normal_scipy_distribution_then_it_should_return_correctly_
 def test_given_gaussian_data_when_fitting_scipy_distribution_automatically_then_it_should_return_correctly_fitted_parameter_values():
     distribution = ScipyDistribution()
 
-    X = np.random.normal(0, 1, 1000)
```

Comment:
Recommend not to introduce many changes in one PR (specially as the change you describe should only concern class renaming).